src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -191,11 +192,17 @@ class Microsoft::Terminal::Core::Terminal final :
     bool _suppressApplicationTitle;
 
 #pragma region Text Selection
-    COORD _selectionStart;
-    COORD _selectionEnd;
-    COORD _selectionPivot;
+    // a selection is represented as a range between two COORDs (start and end)
+    // the pivot is the COORD that remains selected when you extend a selection in any direction
+    //   this is particularly useful when a selection is extended over its starting point
+    struct SelectionAnchors

```

Comment:
Pivot only super makes sense in the context of a word or line selection.

Also, if you draw a big ASCII art diagram here of a few options, it could help someone.